1_html/readme.md

```diff
@@ -28,4 +28,4 @@
 ### Accessibility from day one!
     TODO
 ## Consuming external resources through links
-    TODO
+> The HTML External Resource Link element (<link>) specifies relationships between the current document and an external resource. This element is most commonly used to link to stylesheets, but is also used to establish site icons (both "favicon" style icons and icons for the home screen and apps on mobile devices) among other things. [Mozilla Developers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link)
```

Comment:
Hi! thanks for your contribution, please remove the quote level `>` on this paragraph to clean up the styling.